fontc/src/main.rs

```diff
@@ -35,14 +35,12 @@ fn run() -> Result<(), Error> {
     env_logger::builder()
         .format(|buf, record| {
             let ts = buf.timestamp_micros();
+            let style = buf.default_level_style(record.level());
             writeln!(
                 buf,
-                "[{ts} {} {} {}] {}",
-                // we manually assign all threads a name
```

Comment:
Knowing what thread things are on *is* helpful, I don't know that we should remove that. If the comment is wrong and we arent' naming them anymore lets just fix the comment?

Reply:
Hm. I see we get unknown a lot. Let's make it print the thread id if there is no name, unknown is ofc not useful.

Reply:
it's just annoying because the only way to print a thread id is the full debug version where you get `ThreadId(8)` or whatever, and it takes up a lot of space. will make this change though..